<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>514</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Afiní šifra</string>
  </property>
  <widget class="QRadioButton" name="radioButtonSiforvat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Šifrovat</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonOdsifrovat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Odšifrovat</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVstupniData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vstupní data</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVystupniData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Výstupní data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonProved">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vypočítej</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNacist">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst text ze souboru</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUloz">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit výstup</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditA">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditVstup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditVystup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>2</number>
   </property>
   <property name="columnCount">
    <number>26</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>28</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>28</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Vstup a výstup</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
